<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐播放器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,5,2">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>音乐播放器</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="title">
      <string>搜索</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
      <item>
       <widget class="QLineEdit" name="Edit_input"/>
      </item>
      <item>
       <widget class="QPushButton" name="Bsearch">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="title">
      <string>选择下载网站</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Bnetease">
        <property name="text">
         <string>网易云</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bkugou">
        <property name="text">
         <string>酷狗</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bxiami">
        <property name="text">
         <string>虾米</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bbaidu">
        <property name="text">
         <string>百度</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="title">
      <string>歌曲列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="song_list"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="title">
      <string>控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_state">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Bpause">
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Bup">
          <property name="text">
           <string>音量加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Bdown">
          <property name="text">
           <string>音量减</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
